src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -1407,4 +1425,54 @@ public static void truncate(final List<?> list, final int maxLength) {
             }
         }
     }
+
+    /**
+     * Convert a logical month of the year into the Calendar constant for that month.
+     *
+     * Will throw an {@link IllegalArgumentException} if the given month value falls outside
+     * the expected range - [1-12] inclusive.
+     *
+     * @param month The logical month to convert in the range [1-12] inclusive.
+     * @return The Calendar constant for the given month.  One of:
+     *         {@link java.util.Calendar.JANUARY}, {@link java.util.Calendar.FEBRUARY},
+     *         {@link java.util.Calendar.MARCH}, {@link java.util.Calendar.APRIL}, {@link java.util.Calendar.MAY},
+     *         {@link java.util.Calendar.JUNE}, {@link java.util.Calendar.JULY}, {@link java.util.Calendar.AUGUST},
+     *         {@link java.util.Calendar.SEPTEMBER}, {@link java.util.Calendar.OCTOBER},
+     *         {@link java.util.Calendar.NOVEMBER}, {@link java.util.Calendar.DECEMBER}
+     */
+    public static int getCalendarMonth(final int month) {
+
+        final String baseMessage = "Given month falls outside the acceptable range [1-12]: ";
+
+        Utils.validateArg((1 <= month) && (month <= 12), baseMessage + month);
+
+        switch (month) {
+            case 1:
+                return Calendar.JANUARY;
```

Comment:
You should consider migrating to the newer `java.time.*` classes (eg., `LocalDate`) that were introduced in Java 8, instead of using the legacy `Calendar` class. For example, `java.time` has a proper `Month` enum with a `Month.of(int)` method, which would eliminate the need for a method like this.

Reply:
Yeah - the old way was deprecated and this was the first solution that I found.  I agree - it's terrible.

I'll switch over to `LocalDate` or some equivalent.